faq/client.md

```diff
@@ -0,0 +1,237 @@
+<!--
```

Comment:
Copyright should come after the first header and look like line one here (with the appropriate copyright):

https://raw.githubusercontent.com/Cargill/splinter-docs/master/docs/0.4/index.md

(same applies to all other headers that look like this)